src/pages/profiles/ProfileUsedByProject.tsx

```diff
@@ -6,13 +6,21 @@ import { LxdProfile } from "types/profile";
 import { useQuery } from "@tanstack/react-query";
 import { fetchProjects } from "api/projects";
 import { queryKeys } from "util/queryKeys";
+import classnames from "classnames";
 
 interface Props {
   profile: LxdProfile;
   project: string;
+  headingClassName?: string;
+  hasTableParent?: boolean;
 }
 
-const ProfileUsedByProject: FC<Props> = ({ profile, project }) => {
+const ProfileUsedByProject: FC<Props> = ({
+  profile,
+  project,
+  headingClassName,
+  hasTableParent = false,
```

Comment:
Better avoid this kind of props, that let a component behave in two different ways. The two usages look so similar it should be possible to make this component generic to fit both without a need to distinguish.

Reply:
I think the component is doing too many things. It renders fundamentally different based on default or non-default project. I think this logic might be better pushed into child components and contained there.
As for the table logic - I think we should not even use tables here. Something like a grid or flex layout might be better, as the data is not tabular by nature.

Reply:
The parent in the summary panel is a table, so unless you want to completely change the structure of the summary panel itself (which sounds pretty much out of scope, because it involves changing the instance's one as well for consistency), the child component needs to be wrapped inside a table row.

Apart from that, I tried to follow your suggestions, and simplified the code by creating 2 different children components for the default/non-default cases. Nonetheless, the flag is still needed, because in the summary panel we have the single project list right aligned, whereas in the overview we don't need such thing.